ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/DropSourceCommand.java

```diff
@@ -76,60 +75,44 @@ public DdlCommandResult run(MetaStore metaStore, boolean isValidatePhase) {
         new DropTopicCommand(dataSource.getKsqlTopic().getTopicName());
     metaStore.deleteSource(sourceName);
     dropTopicCommand.run(metaStore, isValidatePhase);
-
     deleteTopicIfNeeded(dataSource, isValidatePhase);
-
     return new DdlCommandResult(true, "Source " + sourceName + " was dropped. "
-                                      + (deleteTopic ? "Topic '"
-                                                    + dataSource.getKsqlTopic().getTopicName()
-                                                    + "' was marked for deletion. Actual deletion "
-                                                    + "and removal from brokers may take some time "
-                                                    + "to complete." : ""));
+        + (deleteTopic ? "Topic '"
```

Comment:
Likewise, you've got different code style for indentation.  Could you correct, please?